### DISCUSSION TITLE
IPGen failed issue

### DISCUSSION THREAD

**Discussion by MFJI** [Xilinx/finn#859] [Q&A] [answered]
I am running FINN0.9 and when it comes to the step "step_hls_ipgen", I got this error:
![image](https://github.com/Xilinx/finn/assets/58033334/e6198246-7868-48e3-91ec-963f967d3bb0)
and I looked into the log file of the mentioned location, I got this:
![image](https://github.com/Xilinx/finn/assets/58033334/e2517a60-34c2-4c91-8cf1-55a85cb0ad8b)
Does anybody know what might cause this problem and how to solve this?
Thank you in advance.

**Comment by fpjentzsch** [comment, marked as answer]
Hi, could you try to follow the steps mentioned in [this forum thread](https://support.xilinx.com/s/question/0D54U00006FYojlSAD/vivado-20222-on-ubuntu-with-error-lcall-cannot-change-locale-enusutf8?language=en_US) and see if it resolves the issue?
